### DISCUSSION TITLE
Suggestions List size for DataSearch

### DISCUSSION THREAD

**Discussion by MarUser04** [appbaseio/reactivesearch#1731] [Q&A] [answered]
Hi all. I have a question about how to increase the size of the suggestion list with the **DataSearch** component search.

I was checking in the documentation of the component and I don't see any prop to increase the size of the list. Currently it returns 20 items in each search. I need to increase it to 100 or some custom value.

I'm using React 17 and Reactivesearch 2.16.1

Thanks in advance for your help!

**Comment by siddharthlatest** [comment, marked as answer]
@MarUser04 Any reason to use ReactiveSearch 2.16.1? I would recommend using the latest ReactiveSearch, our current release is v3.19.1.

I would recommend using the `size` prop to get a different number of suggestions to show. Here's an example of it: https://codesandbox.io/s/crimson-frog-dc7wj?file=/src/index.js.
